Cargo.toml

```diff
@@ -81,3 +81,7 @@ debug = 1
 [patch.crates-io]
 # Waiting on https://github.com/hyperium/headers/pull/99.
 headers = { git = "https://github.com/MaterializeInc/headers.git" }
+# Until https://github.com/jorgecarleitao/parquet-format-rs/pull/2 is merged and released
```

Comment:
So, lesson learned the hard way: fork the repo to the @MaterializeInc repository, then merge your branch to main/master. No need to get a code review, just make sure your change winds up on the MaterializeInc master. Then point the Git dependency at _that_.

Otherwise when you go to update the PR in response to feedback, you'll force-push at the branch without thinking about it, GitHub will helpfully delete the old commits forever, and then Materialize becomes unbuildable for everyone. 🤪 

Also: thank you so much for fighting this fight!

Reply:
Done!